kalite/distributed/management/commands/setup.py

```diff
@@ -50,18 +59,41 @@ def raw_input_password():
     return password
 
 
+def clean_pyc(path):
+    """Delete all *pyc files recursively in a path"""
+    if not os.access(path, os.W_OK):
+        warnings.warn(
+            "{0} is not writable so cannot delete stale *pyc files".format(path))
+        return
+    print("Cleaning *pyc files (if writable) from: {0}".format(path))
+    for root, __dirs, files in os.walk(path):
+        pyc_files = filter(
+            lambda filename: filename.endswith(".pyc"), files)
+        py_files = set(
+            filter(lambda filename: filename.endswith(".py"), files))
+        excess_pyc_files = filter(
+            lambda pyc_filename: pyc_filename[:-1] not in py_files, pyc_files)
+        for excess_pyc_file in excess_pyc_files:
```

Comment:
Based on this line, we don't remove pyc files that have preexisting py files. Is that correct? If so, the docstring needs to reflect that.